app/src/main/java/mod/jbk/build/BuiltInLibraries.java

```diff
@@ -290,19 +300,17 @@ public class BuiltInLibraries {
             new BuiltInLibrary(HTTP_LEGACY_ANDROID_28),
             new BuiltInLibrary(JETBRAINS_ANNOTATIONS),
             new BuiltInLibrary(KOTLIN_STDLIB, List.of(JETBRAINS_ANNOTATIONS)),
-            new BuiltInLibrary(KOTLIN_STDLIB_JDK7, List.of(KOTLIN_STDLIB)),
             new BuiltInLibrary(LOTTIE, List.of(ANDROIDX_APPCOMPAT, OKIO), "com.airbnb.lottie"),
-            new BuiltInLibrary(MATERIAL, List.of(ANDROIDX_ANNOTATION, ANDROIDX_ANNOTATION_EXPERIMENTAL, ANDROIDX_APPCOMPAT, ANDROIDX_CARDVIEW,
+            new BuiltInLibrary(MATERIAL, List.of(ANDROIDX_ANNOTATION, ANDROIDX_ANNOTATION_EXPERIMENTAL, ANDROIDX_ACTIVITY, ANDROIDX_RESOURCEINSPECTION_ANNOTATION, ANDROIDX_APPCOMPAT, ANDROIDX_CARDVIEW,
```

Comment:
That resourceinspection dependency is not useful in sketchware. It's only used by android studio. You can remove it.